app/client/widgets/CellStyle.ts

```diff
@@ -22,11 +22,56 @@ export class CellStyle extends Disposable {
 
   public buildDom(): DomContents {
     return [
+      cssLine(
+        cssLabel(t('HEADER STYLE')),
+      ),
+      cssRow(
```

Comment:
We should only build these when the widget is a table, as header color doesn't apply to card, chart, or custom widgets. To test that the type of the current widget is a table, you should be able to do something like:

```javascript
this._field.viewSection().parentKey() === 'record'
```

Reply:
Thank for that remark, I corrected it and added a test